### DISCUSSION TITLE
what have l done wrong ?

### DISCUSSION THREAD

**Discussion by Abdullah-Mahthel** [LunarVim/LunarVim#4288] [General]
please check these images 

l am trying to install it on windows 10 :/

![lunarissue](https://github.com/LunarVim/LunarVim/assets/130181169/84ab1ed8-0d9c-45b7-a360-39907c8592c3)
![lunarinstallissue](https://github.com/LunarVim/LunarVim/assets/130181169/f03c9cfc-a85c-4687-b12b-f5edd2455f01)

**Comment by LostInTheLogs** [comment]
it looks like your neovim is interpreting lua as vimscript for some weird reason, I'd try reinstalling neovim, maybe a different version

**Reply by Abdullah-Mahthel** [reply]
Windows is a wired operating system
